css/selectors/contains-invalid.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Selectors: selector list containing invalid selector</title>
+    <link rel="help" href="https://drafts.csswg.org/selectors-4/#grouping">
+    <meta name="assert" content="If a selector is invalid, the entire selector list is invalid.">
+    <meta name="assert" content="Pseudo-elements are not valid within :is :not :where.">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+      .a, .b, .c {
+        color: lime;
+      }
+
+      .a, :nonsense {
+        color: red;
+      }
+
+      /* Pseudo-elements cannot be represented by the matches-any pseudo-class; they are not valid within :is(). */
+      .b, :is(::before) {
+        color: red;
+      }
+      .b, :not(::before) {
```

Comment:
Sort of the same comment as https://github.com/web-platform-tests/wpt/pull/14442/files#r240427239, except that there's still ongoing discussion in that issue about how invalidation of `:not()` arguments should work. We should wait until the dust settles on that.